api/tests/Api/MaterialItems/UpdateMaterialItemTest.php

```diff
@@ -151,9 +151,9 @@ public function testPatchMaterialItemValidatesMissingMaterialList() {
             'materialList' => null,
```

Comment:
The following test would be nice:
update a materialitem with a null material list

-> if you set the materiallist to not null -> success
-> if you change another property and leave the materiallist null -> error
-> if you change the material list from null to a material list of another camp -> error